api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -295,8 +269,69 @@
                                   String workspaceId,
                                   Long cohortId,
                                   Long cdrVersionId,
+                                  Long participantId,
                                   ModifyCohortStatusRequest cohortStatusRequest) {
-        return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED)
-                .body(new org.pmiops.workbench.model.ParticipantCohortStatus());
+
+        Cohort cohort = findCohort(cohortId);
+        //this validates that the user is in the proper workspace
+        validateMatchingWorkspace(workspaceNamespace, workspaceId, cohort.getWorkspaceId());
+
+        CohortReview cohortReview = findCohortReview(cohortId, cdrVersionId);
+
+        ParticipantCohortStatus participantCohortStatus = participantCohortStatusDao.
+                findByParticipantKey_CohortReviewIdAndParticipantKey_ParticipantId(
+                        cohortReview.getCohortReviewId(), participantId);
+        if (participantCohortStatus == null) {
+            throw new BadRequestException(
+                    String.format("Invalid Request: No participant exists for participantId: %s", participantId));
+        }
+
+        participantCohortStatus.setStatus(cohortStatusRequest.getStatus());
```

Comment:
One thing we may want to consider in future is setting a timestamp on individual participant_cohort_status rows as well, or even keep a history of status updates. (This would allow us to, for example, perform an analysis on a snapshot of the cohort review even when the cohort review subsequently gets modified.)

(I wouldn't try to do that now, just noting.)